gp-templates/helper-functions.php

```diff
@@ -287,6 +287,100 @@ function textareas( $entry, $permissions, $index = 0 ) {
 	<?php
 }
 
+/**
+ * Render multiple notes
+ *
+ * @param GP_Translation $entry            The translation entry.
+ * @param GP_Glossary    $permissions      Permissions of the user.
+ *
+ * @return void
```

Comment:
When a function has no return value you don't need to add an @return line.

Reply:
Looking at `render_note()`, it has a return value, these two functions should be consistent one way or the other.